### PR TITLE
Refactored code

### PR DESCRIPTION
**Pull request by imanghafoori1**
I try to keep my pull request easy to understand and trust.

### CODE REVIEW

**Comment by imanghafoori1** [comment]
I do not know why the 3 commits  of the lasted PR are listed here.
But I am pretty well sure that they are harmless when merging.

**Comment by imanghafoori1** [comment]
Hope you have had a nice holidays. @balping

File: src/TicketitServiceProvider.php [hunk 1]
```diff
@@ -55,42 +54,13 @@ public function boot()
             TicketItComposer::summerNotes();
 
             // Send notification when new comment is added
-            Comment::creating(function ($comment) {
-                if (Setting::grab('comment_notification')) {
```
Comment (thekordy) [inline src/TicketitServiceProvider.php:59]: @imanghafoori1 I think in this case, you should have moved notification sending to the observers as well, right? or am I missing something?

**Comment by balping** [comment]
@imanghafoori1 The previous commits are visible because you use the same branch. To have clean PRs, use a different branch for each of them

File: src/TicketitServiceProvider.php (hunk 1 above)
Comment (imanghafoori1) [reply src/TicketitServiceProvider.php:59]: if you mean the comments, you are right. I kept them there so the git had a better hint to find out what is replaced with what.
I will remove them in the next pr

**Comment by imanghafoori1** [comment]
I tried an other branch but the same thing happened. so I reopend this PR. I will find a remedy for it

File: src/TicketitServiceProvider.php (hunk 1 above)
Comment (thekordy) [reply src/TicketitServiceProvider.php:59]: I mean this 
```php
if (Setting::grab('comment_notification')) {
    $notification = new NotificationsController();
    $notification->newComment($comment);
}
```
I don't understand how notifications are going to be sent in this PR, I see the new observers classes are empty.

Comment (thekordy) [reply src/TicketitServiceProvider.php:59]: and same for ticket updating and created events as well.

**Comment by balping** [comment]
What's the purpose of this? As @thekordy pointed out, the observer classes are empty. If you need events, why don't just use Laravel's built-in event system?

File: src/TicketitServiceProvider.php (hunk 1 above)
Comment (imanghafoori1) [reply src/TicketitServiceProvider.php:59]: Oh my god, it seems I have lost changes while struggling with the commits issue. I fix it now

**Pull request closed without merging**